Sample Apps/Storefront/Storefront/ClientQuery.swift

```diff
@@ -306,8 +306,8 @@ final class ClientQuery {
             paymentAmount:  paymentAmount,
             idempotencyKey: idempotencyToken,
             billingAddress: mailingAddress,
-            type:           CheckoutViewModel.PaymentType.applePay.rawValue,
-            paymentData:    token
+            paymentData:    token,
+            type:           CheckoutViewModel.PaymentType.applePay.rawValue
```

Comment:
This is a perfect example of such breaking change. We should avoid introducing non-functional changes such as this one.